src/chat/chatter.py

```diff
@@ -369,11 +373,22 @@ def set_color(self):
             color = pcolors.getUserColor(_id, name)
         self.setForeground(QtGui.QColor(color))
 
+    # temp sync api handling with request to fix viewAliases 12.2017
```

Comment:
This info will be visible with git blame, we don't neeed it here.

Reply:
ok, what about a TODO instead?

Reply:
Sounds good.